index.html

```diff
@@ -10226,12 +10226,48 @@ <h3>Value</h3>
 	</section>
 	<section id="enumerated-attribute-values">
 		<h2>Enumerated Attribute Values</h2>
-		<p>When the ARIA attribute definition includes a table enumerating the attribute's allowed <span>values</span>, that attribute is an <a href="https://html.spec.whatwg.org/multipage/common-microsyntaxes.html#enumerated-attribute">enumerated attribute</a>. Each value in the table is a keyword for the attribute, mapping to a state of the same name. When the values table notates one of the values as "(default)", that default value is the <a href="https://html.spec.whatwg.org/multipage/common-microsyntaxes.html#missing-value-default">missing value default</a> and <a href="https://html.spec.whatwg.org/multipage/common-microsyntaxes.html#invalid-value-default">invalid value default</a> for the attribute.</p>
-		<section class="informative" id="enumeration-example">
-			<h3>Example Enumerated Attribute Usage</h3>
-			<p>As noted in <a href="#typemapping">Mapping <abbr title="Accessible Rich Internet Applications">WAI-ARIA</abbr> Value Types to Languages</a>, attributes are included in host languages, and the syntax for representation of enumerated value types is governed by the host language.</p>
-			<p>All enumerated attribute getters and setters use string values, including the boolean-like enumerated <a href="#valuetype_true-false">true/false</a> type.</p>
-			<aside class="example">
+		<p>When the ARIA attribute definition includes a table enumerating the attribute's allowed <span>values</span>, 
+		that attribute is a nullable <a href="https://html.spec.whatwg.org/multipage/common-microsyntaxes.html#enumerated-attribute">enumerated attribute</a>. 
```

Comment:
marcos will want us to change this link to the whatwg spec to be in the respec format

Reply:
I couldn't find a way to link to that. If you know how happy to do so but it didn't seem possible in my efforts.

Reply:
@marcoscaceres can you direct us here? 
Otherwise I'd default to: 
`<a data-cite="html/common-microsyntaxes.html#enumerated-attribute">enumerated attribute</a>`

Reply:
You need to ask the HTML folks to export the definition. Or just send them a quick pull request that adds the data-export.

Reply:
Quick example: https://github.com/whatwg/html/pull/6854/files

Literally just add “export” as an attribute and after they merge it will link 🪄 ✨